core/src/test/java/com/inrupt/client/core/MockHttpService.java

```diff
@@ -133,7 +133,7 @@ private void setupMocks() {
                     .withHeader("User-Agent", equalTo(USER_AGENT))
                     .withRequestBody(matching("Test String 1"))
                     .withHeader(CONTENT_TYPE, containing(TEXT_PLAIN))
-                    .withHeader("Authorization", containing("Bearer token-67890"))
+                    .withHeader("Authorization", containing("Bearer eyJ"))
```

Comment:
For my own understanding, why was this required?

Reply:
This allows us to vary the mock server response based on the presence of an auth token.